### ISSUE TITLE
Use git's tags instead of creating branches

### ISSUE THREAD

**Issue by green-coder** [strangeioc/strangeioc#203]
Hi,

For a new comer, it is non-trivial to find the latest release version to be used:
- Tags only point to old releases.
- The release section only contains old releases.
- The only way to understand what's going on is to find the graph of the commits and notice that some branches have the name "release/...".
- There are way too many branches in the branch list (on the github website) for the new comer to notice that some of them start with "release/...".

I suggest to drop all the release branches and use the git tags instead.

**Comment by JoshFisk** [comment]
Agreed. At a glance, the project looks completely dead, and most people are not going to look into it further.

**Comment by leandromperez** [comment]
Mark and Will are still working on the project?

**Comment by fogAndWhisky** [comment]
We are. So sorry...we've been quietly crazy over at Unity and haven't quite had time to bring Strange over the finish line to v1.

'master' is still the only blessed branch. The 'release' branches are badly named...they should say 'rc' since that's what they are. Actual release will indeed be tagged when it happens. @green-coder is also correct that we need to clean up some obsolete branches.

**Comment by WillzZz** [comment]
Yup, I'm active on google groups, as much as I can. v1 hasn't released because we've found issues in some features and I need them to be fixed before releasing. I also wanted to investigate the console build issue (they work! yay!). I'm crunching on 5.4, so it's really hard for me to make time to move this forward. Thankfully we have an awesome community!

**Comment by strich** [comment]
Hey guys. Just wondering where Strange is currently sitting with development? Whats left for v1? Whats on the roadmap? Does it look like you'll have time to get back to it any time soon?

**Comment by leandromperez** [comment]
+1 !!
we are using it for our current project and we would love to know that it's
still alive!

**Comment by WillzZz** [comment]
Yup, very alive. Marc and I are at GDC and both just finished long haul Unity work. I can't speak for him, but I'll have a little more time to spend on strange now that my team's product is out in the wild (Collaborate). If you have specific features or bug requests, feel free to create issues, post on google groups, or send me an email. I spoke with quite a few users this week and saw some games and apps (and servers!) running it. Good times!

**Comment by leandromperez** [comment]
It's great to hear that!! Thanks!

**Comment by paulgeorges** [comment]
:+1:

**Comment by strich** [comment]
@WillzZz Thanks for the update. I think something I'd love to see here is some solid housekeeping. Its boring I know but I think the project would really benefit from seeing a roadmap to v1.0. If there isn't really much to add, then why not v1.0 it now. Clean up the git tags and branches. Get the project into a better position to lead and allow community contributions.

**Comment by haysclark** [comment]
I would suggest using GitFlow, it's more prescriptive which makes it easier for people to contribute.
